src/components/Sum.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a, b }) => {
+  const res = a + b;
+
+  return (
+    <p>
+      {`Sum of ${a} and ${b} is ${res}`}
+    </p>
+  );
+};
```

Comment:
`res` variable is not really of a much use here - `a + b` is shorter than declaring a variable for it
```suggestion
export const Sum = ({ a, b }) => (
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
);
```